cmd/openshift-tuned.go

```diff
@@ -706,12 +730,14 @@ func changeWatcher() (err error) {
 	}
 
 	// Create a ticker to do a full node/pod labels pull
-	resyncPeriod := pullResyncPeriod()
-	tickerPull := time.NewTicker(time.Second * time.Duration(resyncPeriod))
+	tickerPull := time.NewTicker(time.Second * time.Duration(resyncPeriod.now))
 	defer tickerPull.Stop()
-	glog.Infof("Resync period to pull node/pod labels: %d [s]", resyncPeriod)
+	glog.Infof("Resync period to pull node/pod labels: %d [s]", resyncPeriod.now)
+
+	// When the first pod pull should happen; try to avoid lockstep
+	tuned.podLabelsPullTime = time.Now().Unix() + resyncPeriodPodDefault + getJitter(resyncPeriodPodDefault, 0.3)
```

Comment:
Maybe a good candidate to use a method rather than setting the struct value directly as it's pretty similar in both places.

Reply:
Definitely, code duplication is never a good thing -- missed this one, thank you.  Will change as suggested.